pinot-spi/src/main/java/org/apache/pinot/spi/config/table/ingestion/StreamIngestionConfig.java

```diff
@@ -43,6 +43,12 @@ public class StreamIngestionConfig extends BaseJsonConfig {
   @JsonPropertyDescription("Whether pauseless consumption is enabled for the table")
   private boolean _pauselessConsumptionEnabled = false;
 
+  @JsonPropertyDescription("Enforce consumption of segments in order of segment creation by the controller")
+  private boolean _enforceConsumptionInOrder = false;
+
+  @JsonPropertyDescription("If enabled, Server in-memory tracks the sequence number of all loaded segments")
+  private boolean _trackSegmentSeqNumber = true;
```

Comment:
Since we are using this to determine whether to use IS to calculate the previous segment, suggest making it more explicit (such as _useIdealStateToCalculatePreviousSegment`). It is good practice to have boolean config as `false` by default, which is less error-prone